src/lib/Modal.svelte

```diff
@@ -114,12 +106,13 @@
     right: 0;
     bottom: 0;
     background: rgba(0, 0, 0, 0.5);
-    display: flex;
-    justify-content: center;
-    align-items: center;
   }
 
   .mm-container {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
Sticking with the Flexbox approach would mean having to make `mm-modal` `flex`, which seemed like a more substantial change.

So, I have used a `position: fixed;` approach, instead. This is (at least) as well supported as `flex`, across various browsers.

Reply:
Sounds good to me! `fixed` nicely isolates these from the rest of the content, which is what we want here